teamserver_iam.tf

```diff
@@ -1,5 +1,20 @@
+# Create a role that allows the instance to read its certs from S3.
+module "teamserver_certreadrole" {
+  source = "github.com/cisagov/cert-read-role-tf-module"
+
+  providers = {
+    aws = aws.provisioncertreadrole
+  }
+
+  account_ids      = [local.assessment_account_id]
+  cert_bucket_name = var.cert_bucket_name
+  # Certbot stores wildcard certs in a directory with the name of the
+  # domain, instead of pre-pending an asterisk.
+  hostname = var.email_sending_domain
+}
+
 # Create the IAM instance profile for the Teamserver EC2 server
-# instances
+# instance
```

Comment:
This instance profile can be used by any number of Teamserver instances- guessing that's why we had it pluralized before.  Maybe `instance(s)` would make that clearer?

Reply:
Not sure it makes a difference.  I only changed it because right now only one is supported (unless you assign them all the same domain).  If you really want me to change it back I don't mind doing it.

Reply:
Doesn't make a huge difference to me- I'm fine leaving it as is.